addons/base_vat_autocomplete/models/res_partner.py

```diff
@@ -12,6 +12,8 @@
 
 try:
     import stdnum.eu.vat as stdnum_vat
+    if not hasattr(stdnum_vat, "_country_codes"):
```

Comment:
I would check for "country_codes" instead, and then assign `stdnum_vat.country_codes = stdnum_vat._country_codes`, this saves us a line of diff @ line 54

Reply:
FYI The tend will be to use the latest version of the package and the focus is not to use the wrapper line.
IMHO it is better to use the condition when the package is outdated and not when the package is updated.

Odoo’s people comment me something similar in another case. (I don't remember maybe @xmo-odoo)